private-templates-service/client/src/components/Dashboard/PreviewModal/TemplateInfo/VersionCard/VersionCard.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { Template } from 'adaptive-templating-service-typescript-node';
+
+import {
+  CardManageButton,
+  CardTitle,
+  VersionCardHeader, 
+  VersionCardRowTitle,
+  DateWrapper,
+  VersionCardRow,
+  StatusWrapper,
+  VersionIcon, 
+  VersionWrapper
+} from './styled'
+
+import {
+  Card,
+  CardHeader,
+  CardBody, 
+  StatusIndicator, 
+  Status
+} from './../styled';
+
+
+interface Props {
+  template: Template;
+  templateVersion: string;
+}
+
+
+class VersionCard extends React.Component<Props, {}> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  updatedDate = (updatedAt: string | undefined): string => {
+    const oneDay = 24 * 60 * 60 * 1000;
```

Comment:
Move these outside the function so they aren't calculated every time

Reply:
Fixed, thanks!